src/components/Map.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component, Fragment } from 'react';
+import styled from 'styled-components';
+
+const PortalMap = styled.div`
+  display: flex;
+`;
+
+const Item = styled.div`
+  width: 3rem;
+  line-height: 3rem;
+  text-align: center;
+  border: 2px solid white;
+  border-radius: 50%;
+  margin: 1rem;
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+`;
+
+const Arrow = styled.span`
+  display: flex;
+  align-items: center;
+`;
+
+const mapSize = [1,2,3,4,5];
+
+class Map extends Component {
+  render() {
+    return (
+      <PortalMap>
+        {
+          mapSize.map((item, index) => {
+            return (
+              <Fragment>
```

Comment:
you need to add the variable index as a key or else you'll have lack of unique key  problems